### ISSUE TITLE
Add Wallet list_unspent method

### ISSUE THREAD

**Issue by notmandatory** [bitcoindevkit/bdk-ffi#134]
https://docs.rs/bdk/latest/bdk/wallet/struct.Wallet.html#method.list_unspent

```udl 

dictionary OutPoint {
  string txid;
  u32 vout;
};

dictionary TxOut {
  u64 value;
  string address;
};

[Enum]
interface KeychainKind {
  External;  
  Internal;
}

dictionary LocalUtxo {
  OutPoint outpoint;
  TxOut txout;
  KeychainKind keychain;
};

interface Wallet {
  ...
  sequence<LocalUtxo> list_unspent();
};

```

**Comment by dhruv-1001** [comment]
I was going through this issue, and I have a doubt.
Shouldn't this : 
```
[Enum]
interface KeychainKind {
  External;  
  Internal;
}
```
be this instead ?
```
enum KeychainKind {
  "External",
  "Internal",
};
```
According to this page [here](https://mozilla.github.io/uniffi-rs/udl/enumerations.html).

**Comment by dhruv-1001** [comment]
I would like to work on this issue.

**Comment by dhruv-1001** [comment]
I'll raise a PR soon.

**Comment by zoedberg** [comment]
I'm keeping an eye on this and given there's been no action for a month and I was interested in taking a look anyway, I decided to try and implement this. I ended up with a [PR](https://github.com/bitcoindevkit/bdk-ffi/pull/158), so here it is, hoping it might help push this forward.

**Comment by thunderbiscuit** [comment]
Fixed with #158. Thanks @zoedberg!
